### ISSUE TITLE
key ' ' error

### ISSUE THREAD

**Issue by nixonth** [fivethirtyeight/nfl-elo-game#4]
Hello, 

Hoping someone could help me out with an error. When I delete the three #'s in util.py, I get a key error 
````
Traceback (most recent call last):
  File "eval.py", line 8, in <module>
    Forecast.forecast(games)
  File "...data/forecast.py", line 27, in forecast
    team1, team2 = teams[game['team1']] , teams[game['team2']]
KeyError: ''
````

Line 27 is the last provided here:
````
        for game in games:
            team1, team2 = teams[game['team1']] , teams[game['team2']]
````
I thought it may have something to do with how teams is being defined, so rows 11 - 27 are included below.

````
class Forecast:

    @staticmethod
    def forecast(games):
        """ Generates win probabilities in the my_prob1 field for each game based on Elo model """

        # Initialize team objects to maintain ratings
        teams = {}
        for row in [item for item in csv.DictReader(open("data/initial_elos.csv"))]:
            teams[row['team']] = {
                'name': row['team'],
                'season': None,
                'elo': float(row['elo'])
            }

        for game in games:
            team1, team2 = teams[game['team1']] , teams[game['team2']]
````
Thanks in advance - and apologies if this is a lack of competence more than an issue. New to python, if not to coding in general.

**Comment by jayb** [comment]
Hey @nixonth,

Thanks for the heads-up.

There was a period of time over the weekend where some of the games in the 2017 schedule had an empty team, and this was causing the error. For example, after the Patriots won, but before they knew their opponent in the next game.

I updated the way we create the csv so that won't happen in the future.

**Comment by nixonth** [comment]
Hi Jay,

Thanks for taking a look at this so quickly. I was sure it was due to those missing lines, but I still could not for the life of me find a way to fix it. Working now! Love the work you all do.

Tim

On Jan 16, 2018, at 10:14 AM, Jay Boice <notifications@github.com<mailto:notifications@github.com>> wrote:


Hey @nixonth<https://github.com/nixonth>,

Thanks for the heads-up.

There was a period of time over the weekend where some of the games in the 2017 schedule had an empty team, and this was causing the error. For example, after the Patriots won, but before they knew their opponent in the next game.

I updated the way we create the csv so that won't happen in the future.
